packages/types/src/runner-config.ts

```diff
@@ -2,16 +2,29 @@
 
 import { RunnerContainerConfiguration } from "./sth-configuration";
 
-export type RunnerConfig = {
+type CommonSequenceConfig = {
+    id: string;
+    sequencePath: string;
     name: string;
-    container: RunnerContainerConfiguration;
     version: string;
+}
```

Comment:
This would solve the `packageVolumeId` and the Adapter could come with a typeguard for the data.

```suggestion
    [key: string]: any;
}
```